### DISCUSSION TITLE
A question about this program and Windows LTSC

### DISCUSSION THREAD

**Discussion by kuntsarpyalcin** [Devolutions/UniGetUI#2648] [Q&A] [answered]
Hello!

I have a little question as I couldn't find relevant information. Does UnigetUI work on Windows 10 Enterprise 2021 LTSC/Windows 10 IOT Enterprise 2021?

**Comment by Lockszmith-GH** [comment, marked as answer]
I doubt it has been tested.

For the app to actually work you need the prerequisite libraries to work: .NET and Visual C++ Runtimes.
And of course, the package managers must be able to work as well.

So if this version of windows has a functional winget / scoop / chocolatey etc... it should work.

Are you encountering a specific issue?

**Reply by marticliment** [reply]
Yes. It have been told it works, but WinGet is unreliable

**Reply by JustMyGithub** [reply]
> Yes. It have been told it works, but WinGet is unreliable

What do you mean by "unreliable"?
You need to install some missing components manually in order to get winget to install, but after it was installed I did not notice issues for month now

**Reply by kuntsarpyalcin** [reply]
Thank you so much for your replies! I don't have a specific issue, it was a general question. As much as I see winget has support but is experimental support (link: https://github.com/microsoft/winget-cli/discussions/1956)
And it also seems like chocolatey has full support for LTSC versions (link: https://docs.chocolatey.org/en-us/chocolatey-components-dependencies-and-support-lifecycle/)

I have two more questions:
1. What is the minimum Windows version, also Visual C++ and .NET framework version requirements to install UniGetUI?
2. I saw that there is an option from the command-line to install UniGetUI without winget. But it is also written that it is not recommended. Are there any negative consequences of installing UnigetUI without winget?

Thanks a lot again for good work and taking time to answer. 🙏

**Reply by marticliment** [reply]
that you won't have winget support, which is used to display system installed apps. Just attempt to install it. if it does not work, UniGetUI should resort to bundled winget and should (kinda) work
